ui/tutorial-renderer/src/main/java/org/cru/godtools/tutorial/activity/TutorialActivity.kt

```diff
@@ -183,7 +185,10 @@ class TutorialActivity : BaseActivity<TutorialActivityBinding>(), TutorialCallba
                 setResult(RESULT_OK)
                 finish()
             }
-            R.id.action_training_finish, R.id.action_tips_finish -> finish()
+            R.id.action_training_finish, R.id.action_tips_finish -> {
+                setResult(RESULT_OK)
+                finish()
+            }
```

Comment:
instead of doing this, just move the `R.id.action_tips_finish` id to the previous section where we are already setting OK and finishing.

It doesn't hurt anything moving `R.id.action_training_finish` there as well, but there is no need to move it. So it doesn't matter to me if you consolidate that in as well or not.

Reply:
I was confused by this at first but I see you're just saying to consolidate it to the section above since it's doing the same thing. Will do!